qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -282,94 +385,113 @@ def add(self, other: "FermionicOp") -> "FermionicOp":
         return FermionicOp(
             self._data + other._data,
             max(self.register_length, other.register_length),
-            self.sparse_label or other.sparse_label,
+            self.display_format or other.display_format,
         )
 
-    def to_list(self) -> List[Tuple[str, complex]]:
+    # pylint: disable=arguments-differ
+    def to_list(
+        self,
+        display_format: Optional[str] = None,
+    ) -> List[Tuple[str, complex]]:  # type: ignore
         """Returns the operators internal contents in list-format.
 
+        Args:
+            display_format: If sparse, the label is represented sparsely during output.
+                            if dense, the label is represented densely during output. (default: None)
```

Comment:
```suggestion
                            If dense, the label is represented densely during output.
                            The default value is ``None``, in which case ``self.display_format``
                            will be used.
```

Reply:
So this is basically a way to locally, for this method, override the main display_format property, to which it will otherwise default to for its formatting.

Reply:
Let me propose a better change (will have to do so in a separate discussion to include the line above)